lib/CMakeLists.txt

```diff
@@ -66,23 +66,22 @@ if(BUILD_SHARED_LIBS)
 endif()
 
 # Static library (for unittests because of symbol visibility)
-add_library(${STATIC_LIB} STATIC ${NGHTTP2_SOURCES})
+if(BUILD_STATIC_LIBS)
```

Comment:
reasonable to me, this got removed in:
https://github.com/nghttp2/nghttp2/pull/2064/files#diff-4803677bd280f4dded52c428e3c8ac6c618508dcabcbe880b68be85897c2ad41L68